test/mixin/entity-mixin-test.js

```diff
@@ -0,0 +1,134 @@
+import { afterNextRender } from '@polymer/polymer/lib/utils/render-status.js';
+import 'd2l-fetch/d2l-fetch.js';
+
+describe('d2l-organization-name', () => {
+	var sandbox,
+		component,
+		component2;
+
+	beforeEach(() => {
+		sandbox = sinon.sandbox.create();
+
+		var organizationEntity = {
+			properties: {
+				name: 'Course Name',
+				code: 'SCI100',
+			},
+			links: [{
+				rel: ['https://api.brightspace.com/rels/parent-semester'],
+				href: '/semester.json'
+			},
+			{
+				rel: ['self'],
+				href: '/organization.json'
+			}]
+		};
+		var organizationEntity2 = {
+			properties: {
+				name: 'Course Name 2',
+				code: 'SCI101',
+			},
+			links: [{
+				rel: ['https://api.brightspace.com/rels/parent-semester'],
+				href: '/semester2.json'
+			},
+			{
+				rel: ['self'],
+				href: '/organization2.json'
+			}]
+		};
+		var semesterEntity = {
+			properties: {
+				name: 'Semester Name'
+			},
+			links: [{
+				rel: ['self'],
+				href: '/semester.json'
+			}]
+		};
+		var semesterEntity2 = {
+			properties: {
+				name: 'Semester Name 2'
+			},
+			links: [{
+				rel: ['self'],
+				href: '/semester2.json'
+			}]
+		};
+		sandbox.stub(window.d2lfetch, 'fetch', (input) => {
+			const whatToFetch = {
+				'/organization.json': organizationEntity,
+				'/organization2.json': organizationEntity2,
+				'/semester.json': semesterEntity,
+				'/semester2.json': semesterEntity2
+			};
+			return Promise.resolve({
+				ok: true,
+				json: () => { return Promise.resolve(whatToFetch[input]); }
+			});
+		});
+	});
+
+	afterEach(() => {
+		sandbox.restore();
+	});
+
+	describe('Testing Swap', () => {
```

Comment:
Doesn't have to be this pr. We should be proative in testing all the things i think.  Could save some future grief

Reply:
I totally agree. These tests were well over due and could have saved grief. I need more time in a day.